root/roofit/vectorisedPDFs/SlowRooExponential.h

```diff
@@ -0,0 +1,62 @@
+#include "RooAbsPdf.h"
+#include "RooRealProxy.h"
+
+class RooRealVar;
+class RooAbsReal;
+
+/// A slow implementation of SlowRooExponential that doesn't support batch evaluation.
```

Comment:
```suggestion
/// A slow implementation of RooExponential that doesn't support batch evaluation.
```
To avoid recursion :D